4.LastFM/usageExample.js

```diff
@@ -0,0 +1,35 @@
+var loader1 = new LastfmLoader();
+loader1.query = 'method=chart.getTopArtists&limit=2&page=undefined';
+console.log(loader1);
+loader1.load(function(data) {
+  console.log('Raw loader:');
+  console.log(data);
+});
+
+var loader2 = new TopArtistsLoader(3, 4);
```

Comment:
what I can suggest for you here: what about to pass parameters in load function? in this case you don't need to create objects everytime when you want to send request with another query
`load(params, function(data) {console.log(data})`
where params is `{page: 2, limit: 10}`